bokeh/models/tools.py

```diff
@@ -484,6 +488,59 @@ class SaveTool(Action):
 
     '''
 
+class SaveDataTool(Action):
```

Comment:
Since historically we already use plain "Save" to mean "Save image" we might want to consider if there are names that do not hav "Save" in them e.g. `CSVTool`, `DownloadTool` etc